SparkleDotNET/SUAppcastItem.cs

```diff
@@ -39,6 +39,7 @@ public class SUAppcastItem : IComparable<SUAppcastItem> {
                 ItemDescription = (string)dict.ValueForKey("description");
                 FileURL = new Uri((string)enclosure.ValueForKey("url"));
                 DSASignature = (string)enclosure.ValueForKey("sparkle:dsaSignature");
+                RSASignature = (string)enclosure.ValueForKey("sparkle:rsaSignature");

```

Comment:
lgtm but we'd need a new tool to gen this [using sha and private key] for appcast